### PR TITLE
race desc modifications and stat changes to follow

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

adds SOME amount of lore to some of these guys, finally. not great but better than 'idk my bff jill lmao'. also makes them not literally identical in temrs of stats

## Why It's Good For The Game

variety is the life of spice or smth

### CODE REVIEW

**Comment by Mannybrado** [comment]
male tabaxi just worse, you love to see it

(nobody will notice because there's like 1 tabaxi player)

File: code/modules/mob/living/carbon/human/species_types/furry/lizardfolk.dm [hunk 1]
```diff
@@ -33,8 +32,8 @@
 		OFFSET_NECK_F = list(0,-1), OFFSET_MOUTH_F = list(0,-1), OFFSET_PANTS_F = list(0,0), \
 		OFFSET_SHIRT_F = list(0,0), OFFSET_ARMOR_F = list(0,0), OFFSET_UNDIES_F = list(0,0), \
 		)
-	specstats = list("strength" = 1, "perception" = 1, "intelligence" = -2, "constitution" = 0, "endurance" = 1, "speed" = -1, "fortune" = 0)
-	specstats_f = list("strength" = 0, "perception" = 0, "intelligence" = 1, "constitution" = -1, "endurance" = 0, "speed" = 1, "fortune" = 0)
+	specstats = list("strength" = 2, "perception" = -2, "intelligence" = -2, "constitution" = 0, "endurance" = 1, "speed" = -1, "fortune" = 0)
```
Comment (DeltaFire15) [inline code/modules/mob/living/carbon/human/species_types/furry/lizardfolk.dm:35]: stat totals from 0 to -2 for some reason?

**Comment by bearrrrrrrr** [comment]
oopsie

**Comment by bearrrrrrrr** [comment]
forgot to push my local changes lol

good 2 go

**Pull request merged**